assets/src/edit-story/components/form/dateTime/datePicker.js

```diff
@@ -1,101 +1,49 @@
-// eslint-disable-next-line eslint-comments/disable-enable-pair
-/* eslint-disable header/header */
-/* Disable reason: This file is based on Gutenberg Datepicker */
+/*
+ * Copyright 2020 Google LLC
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     https://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
 
 /**
  * External dependencies
  */
-import moment from 'moment';
-// Needed to initialise the default datepicker styles.
-// See: https://github.com/airbnb/react-dates#initialize
-import 'react-dates/initialize';
-import { DayPickerSingleDateController } from 'react-dates';
-import 'react-dates/lib/css/_datepicker.css';
+import Calendar from 'react-calendar';
+import 'react-calendar/dist/Calendar.css';
 import { useRef, useCallback } from 'react';
 import styled from 'styled-components';
 import PropTypes from 'prop-types';
 import { rgba } from 'polished';
 
-/**
- * Internal dependencies
- */
-import { useConfig } from '../../../app';
-import { TIMEZONELESS_FORMAT } from '../../../constants';
-
 const CalendarWrapper = styled.div`
   min-height: 236px;
   border-top: 1px solid ${({ theme }) => rgba(theme.colors.bg.v0, 0.2)};
 `;
 
 function DatePicker({ currentDate, onChange }) {
-  const { isRTL } = useConfig();
-  const momentDate = currentDate ? moment(currentDate) : moment();
```

Comment:
I wonder if it makes sense to still use `moment` now since Dashboard is using it anyway. Originally the idea was to remove `moment` fully due to its size. An alternative would be to remove it from Dashboard as well, thoughts?

Reply:
I don't have a strong opinion on moment, and I don't know its size. But to clarify: dashboard is a wholly separate from editor and I think it will be even more separate in the future. Likely to split into separate project entirely.

Reply:
Should be around `50kB`, the size.

Will leave it out then as originally planned to enable better separation between dashboard and editor.